cli/ts/commands/deployVkRegistry.ts

```diff
@@ -0,0 +1,27 @@
+import { deployVkRegistry } from "maci-contracts"
+import { existsSync, renameSync } from "fs"
+import { contractAddressesStore, oldContractAddressesStore } from "../utils/constants"
+import { logGreen, success } from "../utils/theme"
+import { banner } from "../utils/banner"
+import { resetContractAddresses, storeContractAddress } from "../utils/storage"
+
+/**
+ * Deploy the vkRegistry contract
+ * @param quiet - whether to print the contract address
+ */
+export const deployVkRegistryContract = async (
+    quiet?: boolean
+): Promise<string> => {
+    if(!quiet) banner()
+    // assume that the vkRegistry contract is the first one to be deployed
+    if (existsSync(contractAddressesStore)) {
+        renameSync(contractAddressesStore, oldContractAddressesStore)
+        resetContractAddresses()
+    }
+
+    // deploy and store the address
+    const vkRegistry = await deployVkRegistry(true)
```

Comment:
it would be helpful to understand how you are triggering this error so I know where to add error handling?
```
npx ts-node build/index.js deployVkRegistry


    ███▄ ▄███▓ ▄▄▄       ▄████▄   ██▓
    ▓██▒▀█▀ ██▒▒████▄    ▒██▀ ▀█  ▓██▒
    ▓██    ▓██░▒██  ▀█▄  ▒▓█    ▄ ▒██▒
    ▒██    ▒██ ░██▄▄▄▄██ ▒▓▓▄ ▄██▒░██░
    ▒██▒   ░██▒ ▓█   ▓██▒▒ ▓███▀ ░░██░
    ░ ▒░   ░  ░ ▒▒   ▓▒█░░ ░▒ ▒  ░░▓
    ░  ░      ░  ▒   ▒▒ ░  ░  ▒    ▒ ░
    ░      ░     ░   ▒   ░         ▒ ░
           ░         ░  ░░ ░       ░
                         ░


Welcome to MACI - Minimal Anti Collusion Infrastructure


[✓] VkRegistry deployed at: 0x8CdaF0CD259887258Bc13a92C0a6dA92698644C0
```

Reply:
well I retried it with your latest update and now it works 🤷‍♂️